### ISSUE TITLE
Demo does not work

### ISSUE THREAD

**Issue by dlinch** [twilio/authy-devise-demo#2]
Hello all,

I'm working to integrate auth-devise into my application, and I keep running into issues. I tried spinning the demo up to see a working implementation to model off, but I can't actually get the demo to work either.

I get to the point of sending off an SMS with a token, and I get a generic "something went wrong" error. I have an active Authy project and the production API key is in my ENV locally in the project. What am I doing wrong?

FWIW, I'm getting to the exact same step in my real project and failing as well.

![Screen Shot 2019-04-15 at 10 33 42 AM](https://user-images.githubusercontent.com/5693184/56149576-1715fd80-5f6a-11e9-9a97-8fc8eb3c4200.png)

EDIT:

I discovered the issue was that the authy api key was not set properly. Granted, I didn't follow the Readme instructions directly, I decided to set a `.env` file with the variable so I would be able to quickly see what my key was and switch it out to test new accounts, and it wasn't picking up the file. I think having a `.env` is an extremely common development pattern, so I opened #3 with the intent that no one else would run into my mistake.

**Comment by philnash** [comment]
Agreed, sorry you came across this issue and thanks for raising the PR.

Did you fix this in your real project too?

**Comment by dlinch** [comment]
@philnash I did, though it was failing for a different reason.
The `authy_request_sms_link` [helper](https://github.com/twilio/authy-devise/blob/master/lib/devise-authy/controllers/view_helpers.rb#L21) is doing some rails magic by creating an anchor link that's actually just a post request under the hood. We have some default routing that physically navigates to the `/request-sms` route when you click that link, so I had to remove those from the views and write some custom JS to run the post with the CSRF token after you press a button on the page instead.
